deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/MethodParser.java

```diff
@@ -3,11 +3,23 @@
 
```

Comment:
Commit description 
**fix(iot-deps):** Fixed the issue by modifying MethodParser class 

I don't see any changes in device

Reply:
No change required on device client side.

Reply:
then please change the commit message to reflect deps

Reply:
done